packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1458,20 +1456,10 @@ function emitEnumMember(context: SdkContext, member: any): Record<string, any> {
   };
 }
 
-function emitSimpleType(
-  context: SdkContext,
-  type: Scalar | IntrinsicType | SdkBuiltInType
-): Record<string, any> {
-  let sdkType: SdkBuiltInType;
-  if (type.kind === "Scalar" || type.kind === "Intrinsic") {
```

Comment:
is this because of Dapeng's change?

Reply:
yeah, his change would cause our compile failure but in our side we only use SdkBuiltInType so I just remove this logic.